examples/survival_analysis/survival_analysis.ipynb

```diff
@@ -28,13 +28,15 @@
    "metadata": {},
```

Comment:
<p>This should be <code>np.exp(idata.posterior["beta"]).mean()</code>,  first operate, then mean.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/96/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>